base_synchro/static/description/index.html

```diff
@@ -23,7 +26,7 @@ <h4 class="oe_slogan oe_mb32">Configure Servers</h4>
                 </div>
                 <div class='oe_span12'>
                     <div class="oe_demo oe_picture oe_screenshot">
-                        <img src="v13_base_synchro_1.png">
+                        <img src="v14_base_synchro_1.png">
```

Comment:
STOP the version naming now onwards. 
it should be base_synchro_1 no v18_*, v19_*